docs: add blog entry for penpal v5 support (#106)
Open-Attestation/documentation-website: website/blog/penpal-v5-decentralized-renderer-react-components.md

---
title: Support for penpal v5
description: Support for penpal v5 in decentralized-renderer-react-components
slug: penpal-v5-decentralized-renderer-react-components
authors:
  - name: Laurent Maillet
    title: Co-creator of OpenAttestation
    url: https://github.com/Nebulis
    image_url: https://avatars.githubusercontent.com/u/503185?s=400&u=f636c79081c55c3dfa431dc249314a02e28d8587
tags: [penpal, decentralized-renderer-react-components]
hide_table_of_contents: false
---

We added support for penpal v5 in [decentralized-renderer-react-components](https://github.com/Open-Attestation/decentralized-renderer-react-components) v3.5.0. You can check the changes in the [pull request](https://github.com/Open-Attestation/decentralized-renderer-react-components/pull/39).

For many months, our library was stuck using Penpal v4 due to a [lack of compatibility between Penapl v4 and Penpal v5](https://github.com/Aaronius/penpal/issues/52). To make it short, it's not possible to have a verifier using Penpal v4 with a renderer using Penpal v5, and vice-versa.

> This version doesn't bring new interfaces or new options. As a user, the new version will be transparent, but it's important to note what's going on under the hood.

Reviewer: ```suggestion
This version doesn't bring new interfaces or new options. As an integrator, the new version is unnoticeable, but it's important to note what's going on under the hood.
```
